packages/manager/modules/billing-components/src/components/commitment/commitment.html

```diff
@@ -139,24 +148,20 @@
             <p
                 data-translate="{{ 'billing_commitment_summary_description_' + $ctrl.model.engagement.commitmentType }}"
                 data-translate-values="{
-                    serviceName: $ctrl.service.name,
+                    serviceName: '<strong>' + $ctrl.service.name + '</strong>',
```

Comment:
Using html tags will not work with `data-translate`. I guess you will have to use `ng-bind-html`.

Reply:
@ganeshkumar1989 the `data-translate-values` accepts this kind of bindings, see this example: https://github.com/ovh/manager/blob/master/packages/manager/modules/product-offers/src/product-offers.html#L61